exercises/numbers/isOdd/isOdd.js

```diff
@@ -36,4 +36,12 @@ if (require.main === module) {
   }
 }
 
+if (num % 2 > 0) {
```

Comment:
This check is correct, but same issue with `isOdd` calling itself. This works, though:

```js
function isOdd(num) {
  return num % 2 > 0;
}
```

This also works:

```js
function isOdd(num) {
  return num % 2 === 1;
}
```

So does this:

```js
function isOdd(num) {
  if (num % 2 === 1) {
    return true;
  } else {
    return false;
  }
}
```